src/main/java/salesforce/ui/pages/opportunity/CreatedOpportunityPage.java

```diff
@@ -20,7 +19,9 @@
 import java.util.Map;
 
 public class CreatedOpportunityPage extends BasePage {
-    private final int sleepTime = 2000;
+    private Map<String, String> headersFields;
+    private WebElementAction webElementAction  = new WebElementAction();
```

Comment:
You don't need to declare the WebElementAction since it is already being inherited from the BasePage. You just need to call its getter method "getWebElementAction()"

Reply:
yeah... I saw this from other pages in the project, that why I implement it that way, but I think that we need to decide how is going to be that part, and do it in all pages, so I guess I will fix it in the next PR :)